docs/Security/CVE-2023-23397/index.md

```diff
@@ -55,7 +55,11 @@ Set-Mailbox -Identity "<UserWhoRunsTheScript>" -ThrottlingPolicy "CVE-2023-23397
 
 ### Prerequisites to run the script for Exchange Online
 
-To run this script in an Exchange Online environment, you need to be a Global Administrator or an Application Administrator. The script will create an application with full access permission on all the mailboxes. To scan cloud mailboxes, you also need to add the Azure-AD module to PowerShell. See [Install AzureAD PowerShell for Graph](https://learn.microsoft.com/powershell/azure/active-directory/install-adv2?view=azureadps-2.0) for more information.
+To run this script in an Exchange Online environment, you need to be a `Global Administrator` or an `Application Administrator`. The script will create an application with full access permission on all the mailboxes.
```

Comment:
Should we add details on being able to run script as owner without needing Global or App admin permissions?

Reply:
Intention is for customers to be aware of that approach - in case they want to avoid the cert management caveats